packages/suite/src/utils/wallet/transactionUtils.ts

```diff
@@ -586,36 +586,37 @@ const numberSearchFilter = (
     amount: BigNumber,
     operator: typeof searchOperators[number],
 ) => {
-    const targetAmount =
-        transaction.targets[0] !== undefined
-            ? getTargetAmount(transaction.targets[0], transaction)
-            : transaction.amount;
-
-    if (!targetAmount) {
-        return false;
-    }
+    const targetAmounts =
```

Comment:
it feels like this fn could be reused for searching without operator as well.

Reply:
Addressed in 3153634